app/merginprojectmodel.h

```diff
@@ -43,6 +43,8 @@ class MerginProjectModel: public QAbstractListModel
 {
     Q_OBJECT
     Q_PROPERTY( QString searchExpression READ searchExpression WRITE setSearchExpression )
+    Q_PROPERTY( int expectedProjectCount READ expectedProjectCount NOTIFY expectedProjectCountChanged )
```

Comment:
Looks like this property is never used. It is only in `updateModel` but we do not store it so I guess this can be removed?